### DISCUSSION TITLE
Hourly ET

### DISCUSSION THREAD

**Discussion by jacob-r-g** [pyet-org/pyet#52] [Ideas]
Is there any way to adapt the library's functions for calculating hourly ET instead of daily?

**Comment by mvremec** [comment]
Hi Jacob,

We haven't done it yet, but this is definitely on our "To do" list.  Which method would you prefer for calculating hourly ET? So far, I've only implemented the FAO-56 and ASCE Penman-Monteith methods for Pandas.Series on a separate private branch. If you're interested, I can share the code with you.

Best,
Matevz

**Reply by nnuxdw** [reply]
Hello, Can pyet now support the calculation of daily PET? Thanks

**Reply by mvremec** [reply]
Hi @nnuxdw, Currently, this is not supported yet. Would FAO-56 alone be sufficient for your needs, or do you require support for other methods as well?

**Reply by nnuxdw** [reply]
Thanks for your fast reply. Actually I'm rerifying a surface-subsurface coupled model to simulate hydrology cycling in a basin, and my ET module runs on hourly time step, which means hourly PET should be an input data. Do you mean FAO-56 can generate hourly PET data? Can I directly use code in testfao56.py or should I do something special? Thanks again.

**Comment by jacob-r-g** [comment]
Hi Matevz,

Thanks for the fast reply and sorry for my delay. FAO-56 is what I was thinking. It would be great to see that code/branch, thanks!

Best,
Jacob

**Comment by nnuxdw** [comment]
Thanks for your fast reply. Actually I'm rerifying a surface-subsurface coupled model to simulate hydrology cycling in a basin, and my ET module runs on hourly time step, which means hourly PET should be an input data. Do you mean FAO-56 can generate hourly PET data? Can I directly use code in testfao56.py or should I do something special? Thanks again.



At 2024-12-16 17:44:16, "MatevzVremec" ***@***.***> wrote:

Hi @nnuxdw, Currently, this is not supported yet. Would FAO-56 alone be sufficient for your needs, or do you require support for other methods as well?

**Comment by Tomasinjo** [comment]
I vote for this as well if it is still on the table.

Hvala za vaše delo.
